src/containers/PluginsContainer.js

```diff
@@ -0,0 +1,116 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import Components from '../components';
+import Actions from '../actions';
+
+import "../styles/Plugins.css";
+
+// PluginsContainer Component
+class PluginsContainer extends Component {
+  componentDidMount() {
+    this.props.getData();
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (this.props.accounts !== nextProps.accounts)
+      nextProps.getData();
+  }
+
+  accountExists(values, account) {
+      for (let i = 0; i < values.length; i++) {
+          const element = values[i];
+          if (element.account === account)
+            return i;
+      }
+      return null;
+  }
+
+  getAccountLabel(account) {
+    if (this.props.allAccounts && this.props.allAccounts.status && this.props.allAccounts.values) {
+        const all = this.props.allAccounts.values;
+        for (let i = 0; i < all.length; i++) {
+            const element = all[i];
+            if (element.roleArn.split(':')[4] === account) {
+                return element.pretty;
+            }
+        }    
+    }
+    return null;
+  }
+
+  groupResultsByAccounts(values) {
+      const res = [];
+      for (let i = 0; i < values.length; i++) {
+          const element = values[i];
+          const existsIndex = this.accountExists(res, element.account);
+          if (existsIndex != null && res[existsIndex] && res[existsIndex].results) {
+              res[existsIndex].results.push(element);
+          } else {
+              const label = this.getAccountLabel.bind(this)(element.account);
+              res.push({
+                  account: element.account,
+                  label,
+                  results: [element]
+              });
+          }
+      }
+      return res;
+  }
+
+  render() {
+    const accounts = [];
```

Comment:
You should add a spinner when waiting for API and error message if there is an issue with API response.